### ISSUE TITLE
issue with sending reactions

### ISSUE THREAD

**Issue by HeroOfSocks** [rubenlagus/TelegramBots#1301]
**Describe the bug**
so i was trying to add setReaction metod to my bot and when im trying to run it i see an error that says: 
`Type must be "emoji" in object: ReactionTypeEmoji(type=emoji, emoji=?) `
idk what exactly is wrong, the `type("emoji")` or my unicode which i use. im new to coding so i think i did it wrong

**To Reproduce**
```
public void setReaction(String emojiValue, Long chatID, int msgID) {
		List<ReactionType> reactionTypeList = new ArrayList<>();
		reactionTypeList.add(ReactionTypeEmoji.builder().type((String)"emoji").emoji(emojiValue).build());
		
		SetMessageReaction setReact = SetMessageReaction.builder()
				.chatId(chatID)
				.messageId(msgID)
				.reactionTypes(reactionTypeList)
				.isBig(false)
				.build();
		
		try {
			execute(setReact);
		} catch (TelegramApiException e) {
			e.printStackTrace();
		}
		}
```

**Expected behavior**
I pass the sender and message ID as well as the unicode "\uD83D\uDC68\u200D\uD83D\uDCBB" to the function and expect to see this 👨‍💻 reaction on my message

**Desktop:**
 - OS: Windows 10
- IDE: Eclipse
- Version:  2023-12 (4.30.0)
- Java JDK 17

**Additional context**
english is not my first language so im sorry for any mistakes

**Comment by everbald** [comment]
the problem is with this validation
```java
        if (ReactionType.EMOJI_TYPE.equals(type)) {
            throw new TelegramApiValidationException("Type must be \"emoji\"", this);
        }
``` 
looks like a bug

**Comment by Chase22** [comment]
Fixed in #1303 will be released wit the next version
